src/Widgets/ThumbnailGalleryWidget/thumbnailGalleryWidget.scss

```diff
@@ -0,0 +1,120 @@
+@import "src/assets/stylesheets/variables";
+@import "src/assets/stylesheets/mixins";
+@import "src/assets/stylesheets/bootstrap/bootstrap";
+
+.gutter0 .gallery-box__wrapper {
+  padding: 0;
+}
+.gallery-box {
+  overflow: hidden;
+  max-height: 800px;
+  max-width: 800px;
+  opacity: 1;
+  @include transition(all 0.5s ease-in-out);
+  @include transition-performance;
+
+  &__wrapper {
+    padding: 0 15px;
+  }
+
+  &__image {
+    @include transition(all 0.5s ease-in-out);
+    @include transition-performance;
+
+    &:before {
+      content: "";
+      display: block;
+      padding-top: 80%;
+    }
+  }
+
+  &__content-wrapper {
+    position: absolute;
+    z-index: 1;
+    opacity: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    /*  display: flex;
+    justify-content: center;
+    align-items: center; */
+    @extend .d-flex;
+    @extend .align-items-center;
+    @extend .justify-content-center;
```

Comment:
Please replace `@extend` rules with mixings (via `@include`)  or a plain css directive. See motivation for this in the bootstrap section above.